cmd/dep/testdata/harness_tests/ensure/empty/case1/final/lock.toml

```diff
@@ -0,0 +1,7 @@
+memo = "63510efb9632ec69c1164ce396d7ebea4ad3884b4fa508373da17226d5a39739"
+
+[[projects]]
+  name = "github.com/sdboyer/deptest"
```

Comment:
Yeah, preferably `go-toml` would let us specify formatting options. I'll double-check that doesn't exist already and see if it's an easy thing to add.